### PR TITLE
Fix for `BaiduNetdisk`

### PR DESCRIPTION
**Pull request by shenzhiming88**
```
*$removeparam=from,domain=sogou.com|baidu.com|jetbrains.com|bandcamp.com|apkpure.com
```
This rule blocks users from saving files in `BaiduNetdisk`.

### CODE REVIEW

File: LegitimateURLShortener.txt [hunk 1]
```diff
@@ -2327,6 +2327,13 @@ $removeparam=height,domain=i-viaplay.com.akamaized.net|media.discordapp.net|tvno
 !#endif
 
 ! ——— Allowlist ——— !
+
```
Comment (iam-py-test) [inline LegitimateURLShortener.txt:2330]: Why is there a space here?

**Comment by iam-py-test** [comment]
Thanks.

File: LegitimateURLShortener.txt (hunk 1 above)
Comment (shenzhiming88) [reply LegitimateURLShortener.txt]: Sorry for the accidental addition of blank lines.

Comment (iam-py-test) [reply LegitimateURLShortener.txt]: Its ok. Errors happen

**Comment by shenzhiming88** [comment]
@iam-py-test I think the other two filters should also add corresponding comments.

**Comment by iam-py-test** [comment]
> 
> 
> @iam-py-test I think the other two filters should also add corresponding comments.

Ok, give me a sec

**Comment by iam-py-test** [comment]
Does https://github.com/DandelionSprout/adfilt/commit/734e3a95b45f7afe192c2b805a1cf39d1f5626f0 look like what you were looking for? I added a link to this and https://github.com/DandelionSprout/adfilt/pull/325

**Comment by shenzhiming88** [comment]
> Does [734e3a9](https://github.com/DandelionSprout/adfilt/commit/734e3a95b45f7afe192c2b805a1cf39d1f5626f0) look like what you were looking for? I added a link to this and #325

Awesome! And what about `@@mailinglist$removeparam=/^utm_/`?

**Comment by iam-py-test** [comment]
> 
> 
> > Does [734e3a9](https://github.com/DandelionSprout/adfilt/commit/734e3a95b45f7afe192c2b805a1cf39d1f5626f0) look like what you were looking for? I added a link to this and #325
> 
> Awesome! And what about `@@mailinglist$removeparam=/^utm_/`?

I have no idea where that came from. I'm pretty sure that was added way before I was working on this, and the commit logs gave me no answers. My best guess is that it is designed to unbreak mailing lists
@DandelionSprout do you know?

**Comment by shenzhiming88** [comment]
It was added in https://github.com/DandelionSprout/adfilt/commit/6dd010c70438a84cae7cf6b683d0d99dc56794ad

**Comment by iam-py-test** [comment]
> 
> 
> It was added in [6dd010c](https://github.com/DandelionSprout/adfilt/commit/6dd010c70438a84cae7cf6b683d0d99dc56794ad)

Ah, thanks! How'd you find that?

**Comment by shenzhiming88** [comment]
> > It was added in [6dd010c](https://github.com/DandelionSprout/adfilt/commit/6dd010c70438a84cae7cf6b683d0d99dc56794ad)
> 
> Ah, thanks! How'd you find that?

You can try `git blame`

**Comment by iam-py-test** [comment]
> 
> 
> > > It was added in [6dd010c](https://github.com/DandelionSprout/adfilt/commit/6dd010c70438a84cae7cf6b683d0d99dc56794ad)
> > 
> > 
> > Ah, thanks! How'd you find that?
> 
> You can try `git blame`

Odd. When I tried that on GH (through the show blame) it showed the wrong commit

**Comment by iam-py-test** [comment]
> However, this commit does not give instructions

Probably something he discovered himself? @DandelionSprout

**Comment by shenzhiming88** [comment]
> > > > It was added in [6dd010c](https://github.com/DandelionSprout/adfilt/commit/6dd010c70438a84cae7cf6b683d0d99dc56794ad)
> > > 
> > > 
> > > Ah, thanks! How'd you find that?
> > 
> > 
> > You can try `git blame`
> 
> Odd. When I tried that on GH (through the show blame) it showed the wrong commit

Yes so I first revert to an older version and then run this command,

**Pull request merged**